packages/theme-chalk/src/drawer.scss

```diff
@@ -45,6 +45,13 @@ $directions: rtl, ltr, ttb, btt;
     font-size: 1rem;
   }
 
+  @include e(footer) {
+    padding: var(--el-drawer-padding-primary);
+    padding-top: 10px;
+    text-align: right;
+    box-sizing: border-box;
```

Comment:
This line of code may not be needed.

Reply:
which line do you mean?
`padding: var(--el-drawer-padding-primary)`?
or `padding-top: 10px;`?

Reply:
![image](https://user-images.githubusercontent.com/21104054/149686721-65e7e9e1-9dc7-4e1e-8ab1-558f6fafaea0.png)
`--el-drawer-padding-primary` is defined in `common/var.scss`

Reply:
> which line do you mean? `padding: var(--el-drawer-padding-primary)`? or `padding-top: 10px;`?

`box-sizing: border-box;`  because there is a common style.

 And is `text-align: right` necessary?

Reply:
you are right, I didn't notice that `box-sizing` was set in `b(drawer)`;
`text-align: right` is necessary,` text-align` default be left